README.md

```diff
@@ -27,19 +27,22 @@
 
 ## About
 
-Firefly is IOTA's new official wallet for Chrysalis.
+Firefly Shimmer is the official wallet for managing your tokens and assets on the Shimmer network.
 
 ## Installation
 
-There are a few options to install Firefly:
+There are a few options to install Firefly Shimmer:
 
-- Download from [official Firefly website](https://firefly.iota.org)
-- Build from source code (see [desktop README](https://github.com/iotaledger/firefly/blob/develop/packages/desktop/README.md))
+-   Download from [official Firefly Shimmer website](https://firefly.iota.org)
+-   Download from the latest [GitHub release](https://github.com/iotaledger/firefly/releases/tag/desktop-2.0.0)
+-   Build from source code (see our handbook guide for ["Environment Setup"](https://iotaledger.github.io/firefly/guides/environment-setup/))
 
 ## Contributing
 
 Help out the Firefly project by filing a [bug report](https://github.com/iotaledger/firefly/issues/new?assignees=&labels=bug&template=bug_report.yml), making a [feature request](https://github.com/iotaledger/firefly/issues/new?assignees=&labels=feat&template=feature_request.md) or opening a [pull request](https://github.com/iotaledger/firefly/pulls/).
-See the [contribution guidelines](https://github.com/iotaledger/firefly/blob/develop/.github/CONTRIBUTING.md) for more.
+If making a code contribution, be sure to checkout the [contribution guidelines](https://github.com/iotaledger/firefly/blob/stardust-develop/.github/CONTRIBUTING.md) beforehand.
```

Comment:
```suggestion
Checkout the [contribution guidelines](https://github.com/iotaledger/firefly/blob/stardust-develop/.github/CONTRIBUTING.md) before making a code contribution.
```

Reply:
Suggest these to matt, this isn't even visible unless in the branch.